specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -71,6 +71,11 @@
           },
           {
             "$ref": "#/parameters/PaginationParameter"
+          },
+          {
+            "in": "query",
+            "name": "aiCapabilities",
```

Comment:
>aiCapabilities [](http://example.com/codeflow?start=21&length=14)

Can you do this with a $filter parameter?

Reply:
Could you please respond to the questions asked by the previous reviewer if you have not resolved them ?